02-microarray/ortholog-mapping_microarray_01_ensembl.Rmd

```diff
@@ -0,0 +1,390 @@
+---
+title: "Ortholog Mapping with `hcop` - Microarray"
+author: "CCDL for ALSF"
+date: "`r format(Sys.time(), '%B %Y')`"
+output:   
+  html_notebook:
+    toc: true
+    toc_float: true
+    number_sections: true
+---
+
+# Purpose of this analysis
+
+This notebook demonstrates how you can use the [`hcop`](https://github.com/stephenturner/hcop) package to perform ortholog mapping for microarray data obtained from refine.bio.
+In this notebook, we will use zebrafish data from refine.bio and annotate it with human gene symbols from HGNC Comparison of Orthology Predictions (HCOP).
```

Comment:
Worth linking to the original source here? https://www.genenames.org/tools/hcop/